mirage/config.js

```diff
@@ -226,9 +247,14 @@ export default function configure() {
       });
     }
 
-    return filtered;
+    return filtered && queryFiltered;
   }));
 
+  this.get('/titles/:id', ({ titles }, request) => {
+    return titles.find(request.params.id);
+  });
+
+
```

Comment:
Looks like this was duplicated by accident

Reply:
Yes - probably an issue during a rebase